# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/jsapi/RemoteTrackSource.h
# Commit: 35d86243cb45
# Full Hash: 35d86243cb45f7e02dd6ec0eea04cd62c2a70196
# Author: Andrew Osmond <aosmond@mozilla.com>
# Date: 2024-09-16 09:14:26
# Regressor Bug: 1908245
# File Overlap Count: 1
# Description:
#   Bug 1908245 - Add support for missing requestVideoFrameCallback parameters. r=media-playback-reviewers,webidl,padenot,saschanaz
#   
#   This patch adds support for exposing the capture time, receive time,
#   processing duration and RTP timestamp parameters on the callback for
#   rVFC for WebRTC related elements.
# ==============================================================================

diff -r 7d143d92bf2a -r 35d86243cb45 dom/media/webrtc/jsapi/RemoteTrackSource.h
--- a/dom/media/webrtc/jsapi/RemoteTrackSource.h	Sun Sep 15 20:03:40 2024 +0000
+++ b/dom/media/webrtc/jsapi/RemoteTrackSource.h	Sun Sep 15 21:49:38 2024 +0000
@@ -52,6 +52,8 @@
 
   SourceMediaTrack* Stream() const;
 
+  const dom::RTCStatsTimestampMaker* GetTimestampMaker() const override;
+
  private:
   virtual ~RemoteTrackSource();
 